From 8c55e763768e31d7f924b78300ce1fbda47bf21b Mon Sep 17 00:00:00 2001
From: Wu Liang feng <wulf@rock-chips.com>
Date: Sun, 5 Jun 2016 21:06:28 +0800
Subject: [PATCH] ARM64: dts: rk3399: remove dwc3 tx-fifo-resize property

The FIFO resizing logic has been removed, and the default
Tx FIFO size is right on rk3399, so remove the property.

Change-Id: Id3cc138e07222d9256a5cc9b1c81a6d7523d05fb
Signed-off-by: Wu Liang feng <wulf@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index 5f2e2ff316c7..e206e3cbf3c7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -434,7 +434,6 @@
 			reg = <0x0 0xfe800000 0x0 0x100000>;
 			interrupts = <GIC_SPI 105 IRQ_TYPE_LEVEL_HIGH>;
 			dr_mode = "otg";
-			tx-fifo-resize;
 			snps,dis_enblslpm_quirk;
 			snps,phyif_utmi_16_bits;
 			snps,dis_u2_freeclk_exists_quirk;
@@ -461,7 +460,6 @@
 			reg = <0x0 0xfe900000 0x0 0x100000>;
 			interrupts = <GIC_SPI 110 IRQ_TYPE_LEVEL_HIGH>;
 			dr_mode = "otg";
-			tx-fifo-resize;
 			snps,dis_enblslpm_quirk;
 			snps,phyif_utmi_16_bits;
 			snps,dis_u2_freeclk_exists_quirk;
-- 
2.35.3

